src/mainboard/protectli/vault_adl_n/variants/vp2430/overridetree.cb

```diff
@@ -0,0 +1,71 @@
+chip soc/intel/alderlake
+	# FSP configuration
+
+	# SATA
+	register "sata_ports_enable[0]" = "1" # Header
+	register "sata_ports_enable[1]" = "1" # M.2 2280
+	register "sata_ports_dev_slp[0]" = "1"
+	register "sata_ports_dev_slp[1]" = "1"
+
+	device domain 0 on
+		device ref igpu on
+			subsystemid 0x8086 0x2212
+			register "ddi_portA_config" = "1" # HDMI on port A
+			register "ddi_portB_config" = "1" # DP on port B
+			register "ddi_ports_config" = "{
+				[DDI_PORT_A] = DDI_ENABLE_HPD | DDI_ENABLE_DDC,
+				[DDI_PORT_B] = DDI_ENABLE_HPD | DDI_ENABLE_DDC,
+				[DDI_PORT_1] = DDI_ENABLE_HPD,
+				[DDI_PORT_2] = DDI_ENABLE_HPD,
+			}"
+		end
+
+		# SATA
+		device ref pcie_rp11 off end
+		device ref pcie_rp12 off end
+		# LAN1
+		device ref pcie_rp2 on
+			register "pch_pcie_rp[PCH_RP(2)]" = "{
+				.flags = PCIE_RP_LTR | PCIE_RP_AER | PCIE_RP_CLK_REQ_UNUSED,
+				.pcie_rp_aspm = ASPM_DISABLE,
+				.PcieRpL1Substates = L1_SS_DISABLED,
+				.clk_src = 0,
+				.clk_req = 0,
```

Comment:
There are no CLKREQ pins connected, please remove all of them,

Reply:
All [removed](https://github.com/Dasharo/coreboot/pull/576/commits/734785c0a950d3048c11ac982335b31262decdca#diff-c6b9e16af2e91a2485033bd2f95d2ab30869f1b652824cdb9eca037d0826bfd2L33)